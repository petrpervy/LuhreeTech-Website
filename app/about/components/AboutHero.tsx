export default function AboutHero() {
  return (
    <section className="relative overflow-hidden bg-[radial-gradient(1200px_600px_at_20%_-10%,#c3ddff33,transparent),radial-gradient(1000px_500px_at_80%_-10%,#ffd3a833,transparent)] px-6 py-16 md:py-24">
      <div className="mx-auto max-w-5xl">
        <p className="text-sm font-semibold text-teal-600">About Luhreetech</p>
        <h1 className="mt-3 text-4xl md:text-5xl font-extrabold tracking-tight text-slate-900">
          Small businesses deserve big-company AI leverage—without complexity
        </h1>
        <p className="mt-4 max-w-2xl text-slate-600">
          We design, deploy, and operate AI automations that work like staff—so you reduce labor costs and scale operations without adding headcount.
        </p>
        <div className="mt-6 flex flex-wrap gap-2">
          {['1–2 week typical pilots','Month-to-month','Human-in-the-loop'].map(t => (
            <span key={t} className="rounded-full bg-white/10 ring-1 ring-white/20 px-3 py-1 text-xs text-slate-600 backdrop-blur">
              {t}
            </span>
          ))}
        </div>
      </div>
    </section>
  )
}
