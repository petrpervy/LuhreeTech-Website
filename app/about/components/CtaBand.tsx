'use client'
import { motion, useReducedMotion } from 'framer-motion'
import Link from 'next/link'

export default function CtaBand() {
  const prefersReduced = useReducedMotion()
  
  return (
    <section className="py-16 md:py-24">
      <div className="mx-auto max-w-5xl px-6">
        <motion.div
          initial={{ opacity: 0, y: 20 }}
          whileInView={{ opacity: 1, y: 0 }}
          viewport={{ once: true, amount: 0.4 }}
          transition={{ duration: prefersReduced ? 0 : 0.4 }}
          className="rounded-2xl bg-white/6 ring-1 ring-white/15 shadow-[0_1px_0_0_rgba(255,255,255,0.25),0_20px_40px_-10px_rgba(10,14,23,0.25)] backdrop-blur-xl hover:bg-white/9 hover:shadow-[0_1px_0_0_rgba(255,255,255,0.25),0_30px_60px_-12px_rgba(10,14,23,0.35)] transition-all duration-300 p-8 text-center"
        >
          <h2 className="text-3xl md:text-4xl font-bold text-slate-900 mb-4">
            Ready for a quick audit?
          </h2>
          <p className="text-lg text-slate-600 mb-8 max-w-2xl mx-auto">
            See exactly where AI can give you hours back every week.
          </p>
          
          <div className="flex flex-col sm:flex-row gap-4 justify-center items-center">
            <Link
              href="/contact#book"
              className="inline-flex items-center justify-center rounded-full px-6 py-3 font-semibold text-white bg-gradient-to-r from-teal-500 to-teal-600 hover:from-teal-600 hover:to-teal-700 shadow-lg hover:shadow-xl transform hover:-translate-y-0.5 transition-all duration-200 focus-visible:ring-2 focus-visible:ring-teal-400/60 focus-visible:outline-none"
            >
              Start Free Audit
            </Link>
            
            <Link
              href="/case-studies"
              className="inline-flex items-center text-teal-600 hover:text-teal-700 font-medium transition-colors focus-visible:ring-2 focus-visible:ring-teal-400/60 focus-visible:outline-none rounded-lg px-3 py-2"
            >
              See case studies
            </Link>
          </div>
        </motion.div>
      </div>
    </section>
  )
}
