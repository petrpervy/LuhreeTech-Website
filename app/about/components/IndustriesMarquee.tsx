'use client'
import { motion, useReducedMotion } from 'framer-motion'
import Link from 'next/link'

const industries = [
  { name: 'Law', example: 'missed-call capture & scheduling', tag: 'law' },
  { name: 'Healthcare', example: 'intake + reminders', tag: 'healthcare' },
  { name: 'Logistics', example: 'dispatch & tracking', tag: 'logistics' },
  { name: 'Retail', example: 'service & orders', tag: 'retail' }
]

export default function IndustriesMarquee() {
  const prefersReduced = useReducedMotion()
  
  return (
    <section className="py-16 md:py-24">
      <div className="mx-auto max-w-5xl px-6">
        <h2 className="text-3xl md:text-4xl font-bold text-slate-900 text-center mb-12">
          Who We Work With
        </h2>
        
        <div className="relative overflow-hidden">
          <motion.div
            animate={prefersReduced ? {} : { x: ['0%', '-50%'] }}
            transition={{ 
              duration: prefersReduced ? 0 : 20,
              repeat: prefersReduced ? 0 : Infinity,
              ease: 'linear'
            }}
            className="flex gap-4 whitespace-nowrap"
            onMouseEnter={() => prefersReduced ? null : undefined}
            onMouseLeave={() => prefersReduced ? null : undefined}
          >
            {/* Duplicate for seamless loop */}
            {[...industries, ...industries].map((industry, index) => (
              <Link
                key={`${industry.name}-${index}`}
                href={`/case-studies?tag=${industry.tag}`}
                className="group flex-shrink-0"
              >
                <div className="rounded-2xl bg-white/6 ring-1 ring-white/15 shadow-[0_1px_0_0_rgba(255,255,255,0.25),0_20px_40px_-10px_rgba(10,14,23,0.25)] backdrop-blur-xl hover:bg-white/9 hover:shadow-[0_1px_0_0_rgba(255,255,255,0.25),0_30px_60px_-12px_rgba(10,14,23,0.35)] transition-all duration-300 p-4 focus-visible:ring-2 focus-visible:ring-teal-400/60 focus-visible:outline-none min-w-[200px]">
                  <h3 className="font-semibold text-slate-900 mb-1">{industry.name}</h3>
                  <p className="text-sm text-slate-600">{industry.example}</p>
                  <div className="mt-2 text-xs text-teal-600 opacity-0 group-hover:opacity-100 transition-opacity">
                    View case studies â†’
                  </div>
                </div>
              </Link>
            ))}
          </motion.div>
        </div>
      </div>
    </section>
  )
}
