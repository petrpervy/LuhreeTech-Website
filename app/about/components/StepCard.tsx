'use client'
import { Search, Beaker, Rocket, Activity } from 'lucide-react'
import { motion, useReducedMotion } from 'framer-motion'

const icons = {
  Search,
  Beaker,
  Rocket,
  Activity
}

interface StepCardProps {
  icon: keyof typeof icons
  title: string
  bullets: string[]
  step: number
}

export default function StepCard({ icon: iconName, title, bullets, step }: StepCardProps) {
  const prefersReduced = useReducedMotion()
  const Icon = icons[iconName]
  
  return (
    <motion.div
      initial={{ opacity: 0, y: 20 }}
      whileInView={{ opacity: 1, y: 0 }}
      viewport={{ once: true, amount: 0.4 }}
      transition={{ duration: prefersReduced ? 0 : 0.4, delay: prefersReduced ? 0 : step * 0.1 }}
      whileHover={{ 
        y: prefersReduced ? 0 : -4,
        transition: { duration: 0.2 }
      }}
      className="group relative"
    >
      {/* Progress rail */}
      <div className="absolute -top-2 left-0 right-0 h-1 bg-gradient-to-r from-teal-500/20 to-transparent rounded-full overflow-hidden">
        <motion.div
          initial={{ width: 0 }}
          whileHover={{ width: '100%' }}
          transition={{ duration: prefersReduced ? 0 : 0.3 }}
          className="h-full bg-gradient-to-r from-teal-500 to-teal-400"
        />
      </div>
      
      <div className="rounded-2xl bg-white/6 ring-1 ring-white/15 shadow-[0_1px_0_0_rgba(255,255,255,0.25),0_20px_40px_-10px_rgba(10,14,23,0.25)] backdrop-blur-xl hover:bg-white/9 hover:shadow-[0_1px_0_0_rgba(255,255,255,0.25),0_30px_60px_-12px_rgba(10,14,23,0.35)] transition-all duration-300 p-6 focus-visible:ring-2 focus-visible:ring-teal-400/60 focus-visible:outline-none">
        <div className="flex items-center gap-3 mb-4">
          <div className="w-10 h-10 rounded-xl bg-teal-500/10 border border-teal-500/20 flex items-center justify-center">
            <Icon className="w-5 h-5 text-teal-600" />
          </div>
          <h3 className="text-lg font-semibold text-slate-900">{title}</h3>
        </div>
        
        <ul className="space-y-2">
          {bullets.map((bullet, index) => (
            <li key={index} className="flex items-start gap-2 text-sm text-slate-600">
              <div className="w-1.5 h-1.5 rounded-full bg-teal-500/60 mt-2 flex-shrink-0" />
              <span>{bullet}</span>
            </li>
          ))}
        </ul>
      </div>
    </motion.div>
  )
}
