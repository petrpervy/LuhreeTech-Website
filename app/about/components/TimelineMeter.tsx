'use client'
import { motion, useReducedMotion } from 'framer-motion'

export default function TimelineMeter() {
  const prefersReduced = useReducedMotion()
  
  return (
    <section className="py-8">
      <div className="mx-auto max-w-5xl px-6">
        <div className="rounded-2xl bg-white/6 ring-1 ring-white/15 shadow-[0_1px_0_0_rgba(255,255,255,0.25),0_20px_40px_-10px_rgba(10,14,23,0.25)] backdrop-blur-xl p-8">
          <div className="text-center">
            <h3 className="text-2xl font-bold text-slate-900 mb-2">
              1–2 weeks → typical pilot → go-live
            </h3>
            <p className="text-sm text-slate-600">
              Depends on integrations & approvals
            </p>
          </div>
          
          {/* Gradient track */}
          <div className="mt-8 relative">
            <div className="h-2 bg-gradient-to-r from-teal-500/20 via-teal-500/40 to-teal-500/20 rounded-full overflow-hidden">
              {/* Pulsing marker */}
              <motion.div
                initial={{ x: '0%' }}
                animate={prefersReduced ? { x: '0%' } : { x: ['0%', '100%', '0%'] }}
                transition={{ 
                  duration: prefersReduced ? 0 : 3,
                  repeat: prefersReduced ? 0 : Infinity,
                  ease: 'easeInOut'
                }}
                className="absolute top-0 w-4 h-2 bg-teal-500 rounded-full shadow-lg"
              />
            </div>
            
            {/* Timeline labels */}
            <div className="flex justify-between mt-3 text-xs text-slate-500">
              <span>Audit</span>
              <span>Prototype</span>
              <span>Pilot</span>
              <span>Go-live</span>
            </div>
          </div>
        </div>
      </div>
    </section>
  )
}
