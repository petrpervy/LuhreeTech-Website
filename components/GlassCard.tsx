import { ReactNode } from 'react'
import { cn } from '@/lib/utils'

interface GlassCardProps {
  children: ReactNode
  className?: string
}

export function GlassCard({ children, className }: GlassCardProps) {
  return (
    <div className={cn("glass hover-lift", className)}>
      {children}
    </div>
  )
}
