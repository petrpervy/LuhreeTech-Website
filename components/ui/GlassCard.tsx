import React from "react";
import { cn } from "@/lib/cn";

export type Tint = "lavender" | "peach" | "teal" | "none";

type Props = React.HTMLAttributes<HTMLDivElement> & {
  tint?: Tint;
  hover?: boolean;
};

export default function GlassCard({ className, tint = "none", hover = true, ...props }: Props){
  const tints = {
    none: "",
    lavender: "bg-tint-lavender",
    peach: "bg-tint-peach",
    teal: "bg-tint-teal",
  } as const;

  return (
    <div
      className={cn(
        "card relative", tints[tint], hover && "transition hover:-translate-y-1 hover:shadow-glow",
        className
      )}
      {...props}
    />
  );
}