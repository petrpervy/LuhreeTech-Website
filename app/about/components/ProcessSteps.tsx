import StepCard from './StepCard'

const steps = [
  {
    icon: 'Search' as const,
    title: 'Audit (30–45 min call)',
    bullets: ['Map workflows', 'Choose high-leverage tasks', 'Define success metrics']
  },
  {
    icon: 'Beaker' as const,
    title: 'Prototype (1–3 days)',
    bullets: ['Narrow scope', 'Connect calendars/telephony', 'Test edge cases']
  },
  {
    icon: 'Rocket' as const,
    title: 'Pilot (3–7 days)',
    bullets: ['Limited rollout', 'Logging + alerts', 'Human fallback']
  },
  {
    icon: 'Activity' as const,
    title: 'Operate (ongoing)',
    bullets: ['Monitor KPIs', 'Iterate prompts', 'Add channels/skills']
  }
]

export default function ProcessSteps() {
  return (
    <section className="py-16 md:py-24">
      <div className="mx-auto max-w-5xl px-6">
        <h2 className="text-3xl md:text-4xl font-bold text-slate-900 text-center mb-12">
          How We Work
        </h2>
        
        <div className="grid md:grid-cols-2 lg:grid-cols-4 gap-6">
          {steps.map((step, index) => (
            <StepCard
              key={step.title}
              icon={step.icon}
              title={step.title}
              bullets={step.bullets}
              step={index}
            />
          ))}
        </div>
      </div>
    </section>
  )
}
