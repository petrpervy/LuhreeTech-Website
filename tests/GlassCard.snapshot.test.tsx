import React from 'react'
import { describe, expect, it } from 'vitest'
import { render } from '@testing-library/react'
import { GlassCard } from '@/components/GlassCard'

describe('GlassCard', () => {
  it('renders consistently', () => {
    const { container } = render(<GlassCard>hello</GlassCard>)
    expect(container.firstChild).toMatchSnapshot()
  })
})


