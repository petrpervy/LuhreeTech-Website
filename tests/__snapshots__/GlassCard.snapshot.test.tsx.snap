// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`GlassCard > renders consistently 1`] = `
<div
  class="rounded-3xl border shadow-soft backdrop-blur-xl bg-white/8 border-white/15 relative overflow-hidden before:pointer-events-none before:absolute before:inset-0 before:rounded-3xl before:border before:border-white/30 after:pointer-events-none after:absolute after:inset-0 after:rounded-3xl after:shadow-[inset_0_0_0_1px_rgba(0,0,0,0.15)]"
>
  hello
</div>
`;
